components/atom/radiobutton/src/index.js

```diff
@@ -0,0 +1,57 @@
+import React, {Fragment} from 'react'
+import PropTypes from 'prop-types'
+
+const AtomRadiobutton = ({
+  id,
+  value,
+  disabled,
+  checked,
+  onChange,
```

Comment:
I think that onChange should be required or just has an empty arrow function as a default value